scripts/linux/publishReleasePackages.sh

```diff
@@ -114,6 +116,10 @@ process_args() {
         elif [ $save_next_arg -eq 10 ]; then
             PMC_RELEASE="$arg"
             save_next_arg=0
+        elif [ $save_next_arg -eq 11 ]; then
+            DISCARD="$arg"
+            IS_PMC_SETUP_ONLY="true"
```

Comment:
Because for some reason in `task: AzureCLI@2` when I do `arguments: --setup-pmc-only`, it gets ignored and `setup-pmc-only` is `false`.

Reply:
Ok. I see we already have another option, `--skip-upload` that follows this same pattern. It would be nice to fix this, but it's a nit. No need to dig deeper for this PR.